### ISSUE TITLE
[Bug]: Chat component accessibility issue 

### ISSUE THREAD

**Issue by thisiskrishna** [microsoft/fluentui#31859]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
import { Chat, ChatMessage, ChatMyMessage } from '@fluentui-contrib/react-chat';
```

### Are you reporting Accessibility issue?

Yes

### Reproduction

  https://microsoft.github.io/fluentui-contrib/react-chat/?path=/story/chat--default 

## Actual Behavior
![Screenshot 2024-06-28 101951](https://github.com/microsoft/fluentui/assets/46826077/0f198ce9-6225-4f2f-a966-63ccc757aa34)

  Focusable content not have  have tabindex="-1" 

## Expected Behavior
  Focusable content not have  have tabindex="-1" or be removed from the DOM
  

### Logs

```shell
Title: WCAG 4.1.2: Ensures aria-hidden elements are not focusable nor contain focusable elements (i:nth-child(4))
Tags: Accessibility, WCAG 4.1.2, aria-hidden-focus

Issue: Ensures aria-hidden elements are not focusable nor contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

Target application: Chat - Default ⋅ Storybook - https://microsoft.github.io/fluentui-contrib/react-chat/?path=/story/chat--default

Element path: #storybook-preview-iframe;.fui-Chat > i:nth-child(4)

Snippet: <i tabindex="0" role="none" data-tabster-dummy="" aria-hidden="true" style="position: fixed; height: 1px; width: 1px; opacity: 0.001; z-index: -1; content-visibility: hidden; top: 0px; left: 0px;"></i>

How to fix: 
Fix all of the following:
  Focusable content should have tabindex="-1" or be removed from the DOM

Environment: Microsoft Edge version 126.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.43.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
Hi! Thanks for running accessibility tests and filing the issue 😊. This is actually a known false positive that we have documented on our [internal know issues wiki](https://dev.azure.com/microsoftdesign/fluent-ui/_wiki/wikis/fluent-ui.wiki/334/Fluent-v9-known-issues?anchor=%60aria-hidden-focus%60-on-elements-with-%60data-tabster-dummy%60). Hope that helps you close your bug!
